README.md

```diff
@@ -53,6 +53,7 @@ Then configure the rules you want to use under the rules section.
 - [`jsx-gridicon-size`](docs/rules/jsx-gridicon-size.md): Enforce recommended Gridicon size attributes
 - [`import-docblock`](docs/rules/import-docblock.md): Enforce external, internal dependencies docblocks
 - [`post-message-no-wildcard-targets`](docs/rules/post-message-no-wildcard-targets.md): Disallow using the wildcard '*' in `postMessage`
+- [`jsx-no-unbounded-this-in-props`](docs/rules/jsx-no-unbounded-this-in-props.md): Disallow unbound `this` in JSX Props
```

Comment:
maybe `jsx-no-unbound-members-in-props`? it's not really about `this` being unbound, it's about members being unbound.